plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -169,17 +171,29 @@ def sweep(self, arguments: List[str]) -> None:
         sampler_class = get_class(samplers[self.optuna_config.sampler.name])
         sampler = sampler_class(seed=self.optuna_config.seed)
 
+        directions: List[str]
+        if isinstance(self.optuna_config.direction, ListConfig):
```

Comment:
This is a better (of course with the proper import):

```suggestion
        if isinstance(self.optuna_config.direction, MutableSequence):
```

Reply:
I applied your suggestion in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/c20054f3bdf22ed6585b4b9d382e9b70d3fbe679. Thank you!